requirements.txt

```diff
@@ -1,42 +1,45 @@
 git+https://github.com/OpenTrons/autobahn-python.git#egg=autobahn-python
 
-argh==0.26.2
-attrs==16.0.0
-cbor==1.0.0
-cffi==1.6.0
-click==6.6
-cryptography==1.4
-idna==2.1
-Jinja2==2.8
-lmdb==0.89
-MarkupSafe==0.23
-mistune==0.7.2
-netaddr==0.7.18
-pathtools==0.1.2
-psutil==4.2.0
-py-ubjson==0.8.4
-pyasn1==0.1.9
-pyasn1-modules==0.0.8
-pycparser==2.14
-Pygments==2.1.3
-PyInstaller==3.2
-PyNaCl==1.0.1
-pyOpenSSL==16.0.0
+pyinstaller==3.1.1
 pyserial==3.1.1
-PyTrie==0.2
 PyYAML==3.11
-requests==2.10.0
-sdnotify==0.3.1
-service-identity==16.0.0
-setproctitle==1.1.10
-shutilwhich==1.1.0
-six==1.10.0
-treq==15.1.0
-Twisted==16.2.0
-txaio==2.5.1
 u-msgpack-python==2.1
-watchdog==0.8.3
-zope.interface==4.2.0
+resource
+datetime
```

Comment:
Isn't date time built into python? Is this a separate datetime lib? If so why doesn't it have a version..